src/bufr/README.md

```diff
@@ -67,6 +67,7 @@ Defines how to read data from the input BUFR file. Its sections are as follows:
               hoursFromUtc: 0  # Optional
           satellite_id:
             query: "*/SAID"
+            type: int64
```

Comment:
Is this done solely for testing purposes because the list of satellite identifiers is just an integer and I would find it wildly crazy if the number of values in this table exceeds the limits of single-precision integers (roughly a billion).